src/aotcompile.cpp

```diff
@@ -2023,7 +2003,7 @@ void jl_get_llvmf_defn_impl(jl_llvmf_dump_t* dump, jl_method_instance_t *mi, siz
             return std::make_pair(M.getDataLayout(), Triple(M.getTargetTriple()));
         });
         jl_codegen_params_t output(*ctx, std::move(target_info.first), std::move(target_info.second));
-        output.world = world;
+        output.min_world = output.max_world = world;
```

Comment:
To be accurate with this fix and how it would compile, it looks like this should be
```suggestion
        output.min_world = ci->min_world;
        output.max_world = ci->max_world;
```

Reply:
```suggestion
        output.min_world = codeinst->min_world;
        output.max_world = codeinst->max_world;
```
Or this change more accurately 'cause we may perform new type inference above.